shell/components/Import.vue

```diff
@@ -98,7 +95,22 @@ export default {
 
         btnCb(true);
 
-        this.rows = res;
+        const parsedRes = [];
+
+        // cleanup metadata.state and stateDescription to not show sub-rows in this table of results
```

Comment:
This is a bit hacky, and there's a `subRows` arg on sortable table that should do the job

Reply:
what do you mean @richard-cox ? is there another way of doing this, especially without modifying the `sortableTable` component?

Reply:
https://github.com/rancher/dashboard/blob/master/shell/components/SortableTable/index.vue#L161

Reply:
https://github.com/rancher/dashboard/blob/master/shell/components/Import.vue#L156-L166

sub-rows is not defined, therefore it's `false`. Nevertheless, we've got this:
https://github.com/rancher/dashboard/blob/master/shell/components/SortableTable/index.vue#L880-L885

Which means if there's a `stateDescription`, it's always shown

Reply:
looks like `subRows` is for manually adding a sub row to present something other than description.

we should add a `subRowDescription` param, defaulted to true, which works on https://github.com/rancher/dashboard/blob/master/shell/components/SortableTable/index.vue#L882